examples/apps/btp-operator/README.md

```diff
@@ -0,0 +1,33 @@
+# README
+
+## Install operator
+
+https://kyma-project.io/docs/kyma/latest/03-tutorials/00-service-management/smgt-01-deploy-service/#documentation-content
+
+```sh
+
+helm install sap-btp-operator https://github.com/kyma-incubator/sap-btp-service-operator/releases/download/v0.1.10/sap-btp-operator-v0.1.10.tgz \
+    --atomic \
+    --create-namespace \
+    --namespace=sap-btp-operator \
+    --set manager.secret.clientid='sb-ed2cad1d-c238-4833-b1be-4bd3cef8d32e!b423|service-manager!b4' \
+    --set manager.secret.clientsecret='Yu1o+q7a6GXbsK2jMveun8tVhtU=' \
+    --set manager.secret.url='https://service-manager.cfapps.eu11.hana.ondemand.com' \
+    --set manager.secret.tokenurl='https://busolaoperatortesting.authentication.eu11.hana.ondemand.com'
+
+kubectl patch deployment -nsap-btp-operator sap-btp-operator-controller-manager --type=json -p '[{ "op": "add", "path": "/spec/template/metadata/annotations", "value": {"sidecar.istio.io/inject": "false"} }]'
```

Comment:
I don't think we should mention BTP operator here, we should focus on more open source (or rather non-SAP...) solutions.